.travis.yml

```diff
@@ -3,6 +3,7 @@ env:
   - VERIFY_USE_PUBLIC_BINARIES=true
 jdk:
   - openjdk8
```

Comment:
Probably we can just remove the java 8 build completely....

Reply:
DCS 😞